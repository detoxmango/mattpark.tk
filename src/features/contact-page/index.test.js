import * as Gatsby from 'gatsby'
import React from 'react'
import renderer from 'react-test-renderer'

import ContactPage from './index'

const createNodeMock= (element) => {
  if (element.type === 'p') {
    return {}
  }
  return null
}

describe("ContactPage", () => {
  it("passes", () => {
    expect(true).toBe(true)
  })
  it("renders correctly", () => {
    const tree = renderer.create(<ContactPage />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})