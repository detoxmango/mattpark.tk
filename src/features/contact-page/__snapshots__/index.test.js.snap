// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactPage renders correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="makeStyles-root-2"
  >
    <h1
      className="MuiTypography-root makeStyles-myName-3 MuiTypography-h1"
    >
      Matt Park
    </h1>
    <div
      className="makeStyles-myInfo-4"
    >
      <a
        className="makeStyles-link-5"
        href="tel:+1201-591-3323"
        target="_top"
      >
        <button
          aria-label="Phone Button"
          className="MuiButtonBase-root MuiIconButton-root makeStyles-iconButton-8 MuiIconButton-colorInherit MuiIconButton-edgeStart"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root makeStyles-icon-9"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M15 12h2c0-2.76-2.24-5-5-5v2c1.66 0 3 1.34 3 3zm4 0h2c0-4.97-4.03-9-9-9v2c3.87 0 7 3.13 7 7zm1 3.5c-1.25 0-2.45-.2-3.57-.57-.1-.03-.21-.05-.31-.05-.26 0-.51.1-.71.29l-2.2 2.2c-2.83-1.44-5.15-3.75-6.59-6.59l2.2-2.21c.28-.26.36-.65.25-1C8.7 6.45 8.5 5.25 8.5 4c0-.55-.45-1-1-1H4c-.55 0-1 .45-1 1 0 9.39 7.61 17 17 17 .55 0 1-.45 1-1v-3.5c0-.55-.45-1-1-1zM5.03 5h1.5c.07.88.22 1.75.45 2.58l-1.2 1.21c-.4-1.21-.66-2.47-.75-3.79zM19 18.97c-1.32-.09-2.6-.35-3.8-.76l1.2-1.2c.85.24 1.72.39 2.6.45v1.51z"
              />
            </svg>
            <h2
              className="MuiTypography-root makeStyles-phoneLink-6 MuiTypography-h2"
            >
              (201) 591 - 3323
            </h2>
          </span>
        </button>
      </a>
      <a
        className="makeStyles-link-5"
        href="mailto:mattparksolutions@gmail.com"
        target="_top"
      >
        <button
          aria-label="Email Button"
          className="MuiButtonBase-root MuiIconButton-root makeStyles-iconButton-8 MuiIconButton-colorInherit MuiIconButton-edgeStart"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root makeStyles-icon-9"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 14H4V8l8 5 8-5v10zm-8-7L4 6h16l-8 5z"
              />
            </svg>
            <h2
              className="MuiTypography-root makeStyles-emailLink-7 MuiTypography-h2"
            >
              mattparksolutions@gmail.com
            </h2>
          </span>
        </button>
      </a>
      <div
        className="makeStyles-icons-10"
      >
        <a
          href="https://github.com/mattparksolutions"
          rel="noopener noreferrer"
          target="_blank"
        >
          <button
            aria-label="GitHub Button"
            className="MuiButtonBase-root MuiIconButton-root makeStyles-iconButton-8 MuiIconButton-colorInherit MuiIconButton-edgeStart"
            disabled={false}
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <span
              className="MuiIconButton-label"
            >
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root makeStyles-icon-9"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 .3a12 12 0 0 0-3.8 23.4c.6.1.8-.3.8-.6v-2c-3.3.7-4-1.6-4-1.6-.6-1.4-1.4-1.8-1.4-1.8-1-.7.1-.7.1-.7 1.2 0 1.9 1.2 1.9 1.2 1 1.8 2.8 1.3 3.5 1 0-.8.4-1.3.7-1.6-2.7-.3-5.5-1.3-5.5-6 0-1.2.5-2.3 1.3-3.1-.2-.4-.6-1.6 0-3.2 0 0 1-.3 3.4 1.2a11.5 11.5 0 0 1 6 0c2.3-1.5 3.3-1.2 3.3-1.2.6 1.6.2 2.8 0 3.2.9.8 1.3 1.9 1.3 3.2 0 4.6-2.8 5.6-5.5 5.9.5.4.9 1 .9 2.2v3.3c0 .3.1.7.8.6A12 12 0 0 0 12 .3"
                />
              </svg>
            </span>
          </button>
        </a>
        <a
          href="https://linkedin.com/in/mattparksolutions"
          rel="noopener noreferrer"
          target="_blank"
        >
          <button
            aria-label="LinkedIn Button"
            className="MuiButtonBase-root MuiIconButton-root makeStyles-iconButton-8 MuiIconButton-colorInherit MuiIconButton-edgeStart"
            disabled={false}
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <span
              className="MuiIconButton-label"
            >
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root makeStyles-icon-9"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
                />
              </svg>
            </span>
          </button>
        </a>
      </div>
    </div>
  </div>
  <div
    className="makeStyles-root-62"
  >
    <form
      autoComplete="off"
      className="makeStyles-form-63"
      onSubmit={[Function]}
    >
      <div
        className="MuiFormControl-root MuiTextField-root makeStyles-inputField-64"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
          data-shrink={false}
          htmlFor="name"
          id="name-label"
        >
          Name
          <span
            aria-hidden={true}
            className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
          >
             
            *
          </span>
        </label>
        <div
          className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            autoFocus={false}
            className="MuiInputBase-input MuiOutlinedInput-input"
            disabled={false}
            id="name"
            name="name"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={true}
            type="text"
            value=""
          />
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-125 MuiOutlinedInput-notchedOutline"
          >
            <legend
              className="PrivateNotchedOutline-legendLabelled-127"
            >
              <span>
                Name
                 *
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
      <div
        className="MuiFormControl-root MuiTextField-root makeStyles-inputField-64"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
          data-shrink={false}
          htmlFor="Email"
          id="Email-label"
        >
          Email
          <span
            aria-hidden={true}
            className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
          >
             
            *
          </span>
        </label>
        <div
          className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            autoFocus={false}
            className="MuiInputBase-input MuiOutlinedInput-input"
            disabled={false}
            id="Email"
            name="email"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={true}
            type="email"
            value=""
          />
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-125 MuiOutlinedInput-notchedOutline"
          >
            <legend
              className="PrivateNotchedOutline-legendLabelled-127"
            >
              <span>
                Email
                 *
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
      <div
        className="MuiFormControl-root MuiTextField-root makeStyles-inputField-64"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
          data-shrink={false}
          htmlFor="Message"
          id="Message-label"
        >
          Message
          <span
            aria-hidden={true}
            className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
          >
             
            *
          </span>
        </label>
        <div
          className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-multiline MuiOutlinedInput-multiline"
          onClick={[Function]}
        >
          <textarea
            aria-invalid={false}
            autoFocus={false}
            className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMultiline MuiOutlinedInput-inputMultiline"
            disabled={false}
            id="Message"
            name="message"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={true}
            rows={8}
            value=""
          />
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-125 MuiOutlinedInput-notchedOutline"
          >
            <legend
              className="PrivateNotchedOutline-legendLabelled-127"
            >
              <span>
                Message
                 *
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
      <label
        className="MuiFormControlLabel-root makeStyles-inputField-64"
      >
        <span
          aria-disabled={false}
          className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-141 MuiCheckbox-root"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={null}
        >
          <span
            className="MuiIconButton-label"
          >
            <input
              className="PrivateSwitchBase-input-144"
              data-indeterminate={false}
              name="privacypolicy"
              onChange={[Function]}
              required={true}
              type="checkbox"
            />
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
              />
            </svg>
          </span>
        </span>
        <span
          className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
        >
          <span
            className="MuiTypography-root makeStyles-privacyPolicyText-65 MuiTypography-caption"
          >
            I understand that Matt will securely hold my data in accordance with their privacy policy.
          </span>
        </span>
      </label>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-66 MuiButton-containedSizeLarge MuiButton-sizeLarge"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="submit"
      >
        <span
          className="MuiButton-label"
        >
          Send
          <span
            className="MuiButton-endIcon MuiButton-iconSizeLarge"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M2.01 21L23 12 2.01 3 2 10l15 2-15 2z"
              />
            </svg>
          </span>
        </span>
      </button>
    </form>
  </div>
</div>
`;
